src/modules/dashboard/components/Dialogs/AddExistingSafeDialog/AddExistingSafeDialog.tsx

```diff
@@ -84,18 +82,17 @@ const AddExistingSafeDialog = ({
         contractAddress: undefined,
       }}
       validationSchema={validationSchema}
-      submit={ActionTypes.COLONY_ACTION_GENERIC}
-      error={ActionTypes.COLONY_ACTION_GENERIC}
-      success={ActionTypes.COLONY_ACTION_GENERIC}
+      submit={ActionTypes.COLONY_ACTION_ADD_EXISTING_SAFE}
+      error={ActionTypes.COLONY_ACTION_ADD_EXISTING_SAFE_ERROR}
+      success={ActionTypes.COLONY_ACTION_ADD_EXISTING_SAFE_SUCCESS}
       transform={transform}
       onSuccess={close}
     >
-      {(status, values: FormikProps<FormValues>) => {
+      {(formProps: FormikProps<FormValues>) => {
         return (
           <Dialog cancel={cancel}>
             <DialogForm
-              {...values}
-              status={status}
+              formProps={formProps}
```

Comment:
just destructure formProps like {...formProps} and then there is no need to call formProps in the DialogForm component, you can just access them directly. THis is how we usually do it.